daedalus/src/Daedalus/BackendApi.purs

```diff
@@ -134,10 +134,10 @@ updateTransaction :: forall eff. CAddress -> CTxId -> CTxMeta -> Aff (ajax :: AJ
 updateTransaction addr ctxId = postRBody ["txs", "payments", _address addr, _ctxIdValue ctxId]
 
 getHistory :: forall eff. CAddress -> Int -> Int -> Aff (ajax :: AJAX | eff) (Tuple (Array CTx) Int)
-getHistory addr skip limit = getR ["txs", "histories", _address addr, show skip, show limit]
+getHistory addr skip limit = getR ["txs", "histories", _address addr, "?skip=",show skip, "&limit=",show limit]
```

Comment:
are you sure this will work? This is using https://github.com/input-output-hk/cardano-sl/blob/d76d195ce5fc651d99272f7dc690fbae523ca444/daedalus/src/Daedalus/BackendApi.purs#L32 under the hood which will produce something like `txs/histories/<address>/?skip=/<skip>/&limit/<limit` ?